scaling_performance/install_practices.adoc

```diff
@@ -0,0 +1,89 @@
+[[scaling-performance-install-best-practices]]
+= Recommended Installation Practices
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+
+[[scaling-performance-preinstalling-dependencies]]
+== Pre-installing Dependencies
+
+A node host will access the network to install any RPMs dependencies, such as
+`atomic-openshift-*`, `iptables`, `docker`, and others. Pre-installing these
```

Comment:
Not sure we need the "and others" here. Sounds vague and mysterious!  The "such as" implies the "and other."
...install and RPMs dependencies, such as `atomic-openshift-*`, `iptables`, and `docker`.